wicket/src/runner.rs

```diff
@@ -205,7 +206,11 @@ impl Runner {
             Action::Redraw => {
                 self.screen.draw(&self.state, &mut self.terminal)?;
             }
-            Action::Update(_component_id) => todo!(),
+            Action::Update(component_id) => {
+                self.wicketd.tx.blocking_send(
+                    wicketd::Request::StartUpdate(component_id),
```

Comment:
Just double-checking terminology - a "component" here identifies a sled/switch/psc and not a component within a sled, right?

Reply:
Correct.